<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIRichTextEdit</class>
 <widget class="QWidget" name="UIRichTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>706</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <widget class="QWidget" name="f_toolbar" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_undo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Undo (CTRL+Z)</string>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgUndo.png</normaloff>:/qt_utility/icons/ui/imgUndo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_redo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Redo</string>
        </property>
        <property name="text">
         <string>Redo</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgRedo.png</normaloff>:/qt_utility/icons/ui/imgRedo.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_cut">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Cut (CTRL+X)</string>
        </property>
        <property name="text">
         <string>Cut</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgCut.png</normaloff>:/qt_utility/icons/ui/imgCut.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_copy">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Copy (CTRL+C)</string>
        </property>
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgCopy.png</normaloff>:/qt_utility/icons/ui/imgCopy.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_paste">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Paste (CTRL+V)</string>
        </property>
        <property name="text">
         <string>Paste</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgPaste.png</normaloff>:/qt_utility/icons/ui/imgPaste.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_link">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Link (CTRL+L)</string>
        </property>
        <property name="text">
         <string>Link</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgLink.png</normaloff>:/qt_utility/icons/ui/imgLink.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="f_paragraph">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Paragraph formatting</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="f_fontsize">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Font size</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnIncreaseFont">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgIncreaseFontSize.png</normaloff>:/qt_utility/icons/ui/imgIncreaseFontSize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnDecreaseFont">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgDecreaseFontSize.png</normaloff>:/qt_utility/icons/ui/imgDecreaseFontSize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_bold">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string notr="true">Bold (CTRL+B)</string>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgBold.png</normaloff>:/qt_utility/icons/ui/imgBold.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_italic">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Italic (CTRL+I)</string>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgItalic.png</normaloff>:/qt_utility/icons/ui/imgItalic.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_underline">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Underline (CTRL+U)</string>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgUnderline.png</normaloff>:/qt_utility/icons/ui/imgUnderline.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_strikeout">
        <property name="text">
         <string>Strike Out</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgStrikeThrough.png</normaloff>:/qt_utility/icons/ui/imgStrikeThrough.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_list_bullet">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Bullet list (CTRL+-)</string>
        </property>
        <property name="text">
         <string>Bullet list</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgTextUnordered.png</normaloff>:/qt_utility/icons/ui/imgTextUnordered.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_list_ordered">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Ordered list (CTRL+=)</string>
        </property>
        <property name="text">
         <string>Ordered list</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgTextOrdered.png</normaloff>:/qt_utility/icons/ui/imgTextOrdered.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_indent_dec">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Decrease indentation (CTRL+,)</string>
        </property>
        <property name="text">
         <string>Decrease indentation</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgIndentLess.png</normaloff>:/qt_utility/icons/ui/imgIndentLess.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_indent_inc">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="toolTip">
         <string>Increase indentation (CTRL+.)</string>
        </property>
        <property name="text">
         <string>Increase indentation</string>
        </property>
        <property name="icon">
         <iconset resource="../resx/resources.qrc">
          <normaloff>:/qt_utility/icons/ui/imgIndentMore.png</normaloff>:/qt_utility/icons/ui/imgIndentMore.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="f_image">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="f_menu">
        <property name="text">
         <string>...</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>line_4</zorder>
     <zorder>f_undo</zorder>
     <zorder>f_redo</zorder>
     <zorder>f_cut</zorder>
     <zorder>f_copy</zorder>
     <zorder>f_paste</zorder>
     <zorder>line</zorder>
     <zorder>f_link</zorder>
     <zorder>line_3</zorder>
     <zorder>f_italic</zorder>
     <zorder>f_underline</zorder>
     <zorder>line_2</zorder>
     <zorder>line_5</zorder>
     <zorder>f_list_bullet</zorder>
     <zorder>f_list_ordered</zorder>
     <zorder>f_indent_dec</zorder>
     <zorder>f_indent_inc</zorder>
     <zorder>f_bold</zorder>
     <zorder>f_strikeout</zorder>
     <zorder>f_image</zorder>
     <zorder>line_6</zorder>
     <zorder>f_menu</zorder>
     <zorder>f_fontsize</zorder>
     <zorder>m_btnIncreaseFont</zorder>
     <zorder>line_7</zorder>
     <zorder>m_btnDecreaseFont</zorder>
     <zorder>f_paragraph</zorder>
    </widget>
   </item>
   <item>
    <widget class="cbtek::common::qt_utility::UITextEdit" name="f_textedit">
     <property name="autoFormatting">
      <set>QTextEdit::AutoNone</set>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Text Foreground:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QToolButton" name="m_btnTextBackground">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Text background color</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text Background:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Page Background:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="m_btnTextForeground">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>Text foreground color</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QToolButton" name="m_btnPageBackground">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>cbtek::common::qt_utility::UITextEdit</class>
   <extends>QTextEdit</extends>
   <header>qt_utility/inc/UITextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>f_textedit</tabstop>
  <tabstop>f_strikeout</tabstop>
  <tabstop>f_image</tabstop>
  <tabstop>f_menu</tabstop>
 </tabstops>
 <resources>
  <include location="../resx/resources.qrc"/>
 </resources>
 <connections/>
</ui>
